packages/editor-ui/src/components/NodeExecuteButton.vue

```diff
@@ -168,11 +177,20 @@ export default defineComponent({
 				return this.$locale.baseText('ndv.execute.listenForTestEvent');
 			}
 
+			if (this.isFormTriggerNode) {
+				return this.$locale.baseText('ndv.execute.testStep');
```

Comment:
maybe we can just roll this out globally already? https://linear.app/n8n/issue/ADO-129/update-copy-execute-test

Reply:
this is up to product to decide